package.json

```diff
@@ -89,7 +91,8 @@
     "lint-staged": "^8.0.4",
     "mkdirp": "^0.5.1",
     "mocha": "5.2.0",
-    "npm-run-all": "^4.1.5",
+    "mocha-junit-reporter": "^1.18.0",
+    "npm-run-all": "^4.1.3",
```

Comment:
Looks like a merge hiccup: `npm-run-all` in this repo is now up to 4.1.5, not 4.1.3.